templates/base.html

```diff
@@ -71,7 +70,12 @@
                             </div>
                             <div class="col-sm-6">
                                 <div class="float-right">
-                                    {% block header_button %}{% endblock %}
+                                    {% if extra_html_button_path %}
```

Comment:
what is the difference with `html_button_path` already used in other parts? Do we need both?

Reply:
TODO: remove this block from generic edit
```
{% block header_button %}
    {% if html_button_path %}
        <div class="float-right">
            {% include html_button_path %}
        </div>
    {% endif %}
{% endblock %}
```

And use only extra_html_button_path. Rename html_button_path --> extra_html_button_path